package/kiwi-revert-bls-default-for-suse.patch

```diff
@@ -1,67 +1,57 @@
-From b0d850842b0c8d4c98f0486f66ffd7c6e8f1a685 Mon Sep 17 00:00:00 2001
-From: Neal Gompa <ngompa@opensuse.org>
-Date: Thu, 2 May 2024 06:23:45 -0400
-Subject: [PATCH] Revert "kiwi/bootloader: restore backward compatibility for
- grub2 with bls"
-
-For the time being, SUSE distributions cannot handle KIWI's default to use
-BLS with GRUB2. Until they catch up, revert this.
-
-This reverts commit 155d969d7752d92c90c69044bdd69be70f0a1c24.
```

Comment:
Can we preserve the metadata here so that people know why this patch exists?

Reply:
yes done